prow/config/jobs/istio.yaml

```diff
@@ -36,6 +36,18 @@ jobs:
       - name: TEST_SELECT
         value: "-postsubmit,-flaky,-multicluster"
 
+  - name: integ-cni-k8s-tests
+    types: [presubmit]
+    command: [entrypoint, prow/integ-suite-kind.sh, test.integration.pilot.kube.presubmit]
```

Comment:
will we remove `tests/integration/pilot/cni` then?

Reply:
Yeah I think we can remove the happy path testing, but I think we'd still need dedicated CNI testing on edge cases such as repair.